Cognigy/WebchatWidget
# feature/21182 add data option

This PR adds a data payload to the get started message.

How to test:

1. Create a flow with a say node "data is: {{ci.data.gsdata}}"
2. Create a webchat endpoint for it and embed it into a testing page
3. Add these parameters to the webchat:
startBehavior: "button",
getStartedText: "Get Started",
getStartedPayload: "GET_STARTED_PL",
getStartedData: {"gsdata": "hello world"}
4. Start the testing page with a fresh userid and sessionid and press the welcome button, it should return "data is: hello world"
5. Change startBehavior to "inject", reset userid and sessionid and start it again, it should automatically return the same output
